devdocs/orientation-guide.md

```diff
@@ -0,0 +1,131 @@
+
+<center>
+
+![RolesAndTechnologies.png](https://lh6.googleusercontent.com/3cvr4dEJEBiqlsY_X6-sTwTyYCvk_d5YKsLM2Z-IWI7AEd6c1bYgZ4LDW5yDbwrd9b3kIrGD08kIKyVkg9vHmO-_iXhs8bpGQyJnwpKA12pFWj1Tj-XYuXVYkOBgoI_BVBHkR6I)
+
+</center>
+
+Roles:
+
+* **Tester**: Tests the App manually and reports bugs or suggestions for enhancements in the issue tracker.
+*   **Copy editor**: Helps in improving documentation.
+*   **Web page developer**: Works on static web pages, such as those used in the TEAMMATES web site
+*   **Test developer**: Works on automating system tests.
+*   **Frontend developer**: Works on the frontend of the App that is generated dynamically from the server side.
+*   **Backend developer**: Works on the backend logic of the App, including data storage.
+
+## Workload
+
+[TEAMMATES community](https://www.google.com/url?q=https://teammatesv4.appspot.com/about.html&sa=D&ust=1458427634375000&usg=AFQjCNF9iNnM6PPTk5wwrsnKRUCsLst1xQ) has three tiers, defined based on the level of involvement.
+
+*   Contributor: Small but noteworthy contribution to the project.
+*   Committer: Significant contributions sustained over a long period of time.
+*   Core member: Significant and sustained contributions in recent times. Core members can progress through the levels: Snr Developer, Area Lead, Project Lead, and Project Manager.
+
+## Getting started
+
+Contributing to an OSS project requires you to figure out things on your own when you can, and seek help from the right resource when you cannot. To become a TEAMMATES contributor, you need to start honing those skills. To help you with that, we have created a sequence of tasks you can try to complete.  Try to complete as many of them as you can, in the order they are listed. Of course we are happy to guide you if you encounter any difficulties when doing these tasks; just post your question in [our issue tracker](https://www.google.com/url?q=https://github.com/TEAMMATES/teammates/issues&sa=D&ust=1458427634380000&usg=AFQjCNEfiA-pDqfeFk-NWy7uSU2vJdd4UA). If you are seeking help on project setup, remember to include the following in your post.
+
+1. Step in the setting up guide that you are in
+
+2. What measures you took to address the issue
+
+3. Operating system
+
+4. Screenshots (if any)
+
+
+Note: The task descriptions are brief by intention. We want you to try to figure out how to do those things by yourself.
+
+## Orientation task list
+
+_**Phase A**: Know the project_
+<br />
+1.  Understand what TEAMMATES is about.
+    * [product intro page](https://www.google.com/url?q=https://teammatesv4.appspot.com&sa=D&ust=1458427634385000&usg=AFQjCNEDi3V_xRJj6Sv4x0rTuMreo4zIXA) shown to potential users
+
+    * [project vision, challenges, and principles](https://www.google.com/url?q=http://teammatesv4.appspot.com/dev/devman.html%23project&sa=D&ust=1458427634387000&usg=AFQjCNGm_T3iGMuSsidiGRVwLGI_Ybs4Gg)
```

Comment:
Somehow this text is not appearing properly when I try to view the whole file... @wkurniawan07 any ideas what the problem is?